blocks/README.md

```diff
@@ -129,6 +129,7 @@ Registers a new block provided a unique slug and an object defining its behavior
 - `save( attributes: Object ): WPElement` - Returns an element describing the markup of a block to be saved in the published content. This function is called before save and when switching to an editor's HTML view.
 - `controls: string[]` - Slugs for controls to be made available to block. See also: [`wp.blocks.registerControl`](#wpblocksregistercontrol-slug-string-settings-object-)
 - `encodeAttributes( attributes: Object ): Object` - Called when save markup is generated, this function allows you to control which attributes are to be encoded in the block comment metadata. By default, all attribute values not defined in the block's `attributes` property are serialized to the comment metadata. If defined, this function should return the subset of attributes to encode, or `null` to bypass default behavior.
+- `isVisible: boolean` - Whether the block is to be made available as an option in the block inserter overlay. Defaults to `true`.
```

Comment:
I would prefer `isDeprecated` for deprecating blocks and I don't think we should make this block invisible in the inserter (so maybe the `isVisible` is not worth it for now). IMO the fallback block should be something close to the current editor which will serve as a backwards compatibility solution (think current editor inside as a block).